BUILD.bazel

```diff
@@ -782,15 +808,54 @@ pyx_library(
     name = "_raylet",
     srcs = glob([
         "python/ray/__init__.py",
+        "python/ray/_raylet.pxd",
         "python/ray/_raylet.pyx",
         "python/ray/includes/*.pxd",
         "python/ray/includes/*.pxi",
     ]),
-    copts = COPTS,
+    # Expose ray ABI symbols, which can then be used by _streaming.so.
+    # We need to dlopen this lib with RTLD_GLOBAL to use ABI in this
+    # shared lib, see python/ray/__init__.py.
+    # Note: Ray ABI is not finalized, the exact set of exported (C/C++) APIs
+    # is subject to change.
```

Comment:
Nit, this comment this duplicated with the one in `ray_exported_symbols.lds`. can remove it.

Reply:
removed